### PR TITLE
Update for scalaz 7.1 (days 0 to 10)

### PR DESCRIPTION
**Pull request by js1972**
Minor updates... A couple of syntax errors, typo's and changes from scalaz 7.0 to 7.1. Will get onto the final chapters (days) soon.  ;-)

### CODE REVIEW

File: docs/01/a.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
   [tt]: http://learnyouahaskell.com/types-and-typeclasses
   [moott]: http://learnyouahaskell.com/making-our-own-types-and-typeclasses
   [z7]: https://github.com/scalaz/scalaz/tree/scalaz-seven
-  [z7docs]: http://docs.typelevel.org/api/scalaz/stable/7.0.4/doc
+  [z7docs]: http://docs.typelevel.org/api/scalaz/
```
Comment (xuwei-k) [inline docs/01/a.md:4]: https://oss.sonatype.org/service/local/repositories/releases/archive/org/scalaz/scalaz_2.11/7.1.0/scalaz_2.11-7.1.0-javadoc.jar/!/index.html#scalaz.package

Comment (js1972) [inline docs/01/a.md:4]: This changes the url to point to the one provided on the scalaz gitgub readme which links you to all the documentation versions so you can choose 7.0, 7.1, 7.2.? etc.

Comment (js1972) [inline docs/01/a.md:4]: Can simply change it to the stable 7.1 doco if you want... Just thought this different link would be more useful...

Comment (xuwei-k) [inline docs/01/a.md:4]: http://scalaz.github.io/scalaz/#scaladoc

File: docs/04/c.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ trait Foldable[F[_]] { self =>
 }
 ```
 
-Here are [the operators](https://github.com/scalaz/scalaz/blob/scalaz-seven/core/src/main/scala/scalaz/syntax/FoldableSyntax.scala):
+Here are [the operators](https://github.com/scalaz/scalaz/blob/series/7.1.x/core/src/main/scala/scalaz/syntax/FoldableSyntax.scala):
```
Comment (eed3si9n) [inline docs/04/c.md]: Maybe we should use docs/template.properties for the Github repository's base URL. See [template.properties](http://pamflet.databinder.net/Template+Properties.html).

Comment (js1972) [inline docs/04/c.md]: Good point. will take a look this afternoon... this is the first time I've come across Pamflet and I must say, its pretty cool..!

**Comment by js1972** [comment]
Okay, changed all the links to use a property as suggested. Manually tested all the links as well which is painful as Pamflet runs like a dog on my macbook air!
You might want to have a look at Day14 and if any of it still applies to scalaz 7.1+. It where you put the details of your PR to get Vector included in Scalaz._ etc.

**Comment by js1972** [comment]
If you want to check the changes I've posted it up on my own url for testing at scalaz.jaylin.com.au. I'll continue from day 11 onwards till the end, updating for scalaz 7.1. I've also added a page to discuss the endo type. I haven't commmitted anything else to this PR branch though unless you are interested...  ;-)

**Comment by js1972** [comment]
Howdy... I've made further changes in my own forked branch and have temporarily hosted it at scalaz.jaylin.com.au if you want to check it out. You can see a compare at https://github.com/js1972/learning-scalaz/compare/additions_to_7.1.x
They are all just minor changes to your great resource  ;-)   Adding in some quick links between relevant sections that I found I was always looking up and also adding in some info on the Endo typeclass...
So if you think these additions are worthwhile I can send a PR (I changed the readme for by benefit but I can remove that).

**Comment by eed3si9n** [comment]
Finally I got around to publishing whatever that got merged from this PR - http://eed3si9n.com/learning-scalaz/
The older content is now on - http://eed3si9n.com/learning-scalaz/7.0

As per other content goes, I would accept PRs for typos and factual mistakes, but I'd avoid new materials since the primary purpose is for me to learn Scalaz. If you have a list of things I should look into and expand, I'll welcome that too. Maybe I should ask @xuwei-k what I should read next.

**Comment by js1972** [comment]
ok that's fair enough too... I'll just keep that fork of mine as my own
copy that I can add too myself. The areas you might want to look at (that I
added) are around more examples and use cases of Kliesli as well as
examples of Endo.

;-)

**Pull request merged**